### ISSUE TITLE
Compilation issue with clang

### ISSUE THREAD

**Issue by tristan0x** [ibaned/omega_h#257]
Hello,

With clang@8.0.0-apple, I have the following issue:

```
omega_h-9.15.0/src/Omega_h_hypercube.hpp:6:5: error: unknown command tag name [-Werror,-Wdocumentation-unknown-command
```

Workaround is to disable this check:
```diff
diff --git a/cmake/bob.cmake b/cmake/bob.cmake
index 92bce7cd..0c96bbfa 100644
--- a/cmake/bob.cmake
+++ b/cmake/bob.cmake
@@ -176,6 +176,7 @@ function(bob_begin_cxx_flags)
         set(FLAGS "${FLAGS} -Wno-padded")
         set(FLAGS "${FLAGS} -Wno-float-equal")
         set(FLAGS "${FLAGS} -Wno-weak-template-vtables")
+        set(FLAGS "${FLAGS} -Wno-documentation-unknown-command")
       endif()
       if(APPLE)
         if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "9.0.0")
```

But if you want compilationto fail if unknown Doxygen commands are used, then another solution is to use `-fcomment-block-commands=<arg>,<arg2>` and specify all commands used.

**Comment by ibaned** [comment]
@tristan0x can you check whether 95466d0621ca03a805bb9b3cd3ab2e03782771a8 fixes this?

**Comment by tristan0x** [comment]
Nope. It requires this:

```diff
diff --git a/cmake/bob.cmake b/cmake/bob.cmake
index 56939f5..e3157df 100644
--- a/cmake/bob.cmake
+++ b/cmake/bob.cmake
@@ -178,7 +178,8 @@ function(bob_begin_cxx_flags)
         set(FLAGS "${FLAGS} -Wno-weak-template-vtables")
       endif()
       if(APPLE)
-        if(CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL "8.0.0")
+        if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "8.0.0")
+          message(ERROR "bla")
           set(FLAGS "${FLAGS} -fcomment-block-commands=file")
         endif()
       else()
```

In my case, `CMAKE_CXX_COMPILER_VERSION` equals "8.0.0.8000042"

**Comment by ibaned** [comment]
Ah okay, thanks! I'll make that change

**Comment by ibaned** [comment]
This should be fixed as of `v9.17.1`
